.github/actions/runleaks/.github/workflows/scan_public.yml

```diff
@@ -0,0 +1,61 @@
+name: Scan Public Logs
+
+on: 
+  workflow_dispatch:
+    inputs:
+      repos:
+        description: 'Repos to scan:'
+        required: true
+        default: '[\"josiahsiegel\/runleaks\",\"josiahsiegel\/AzViz-action\"]'
+        type: string
+  schedule:
+    - cron: "0 3 * * *"
+  push:
+
+jobs:
+  pre_job:
+    name: "Set Build Environment"
+    runs-on: ubuntu-latest
+    outputs:
+      repos: ${{ env.REPOS }}
+    steps:
+      - name: Set workflow_dispatch matrix
+        if: github.event_name == 'workflow_dispatch'
+        run: echo "REPOS=${{ github.event.inputs.repos }}" >> $GITHUB_ENV
+
+      - name: Fetch random repo name
+        if: github.event_name == 'schedule' || github.event_name == 'push'
+        uses: JosiahSiegel/randomrepo@v1.1
```

Comment:
needs update

Reply:
All these issues are not documented and we are in the process of reaching out to Mr. Josiah Siegel and ask for his assistance in figuring out the different environment and secrets we need to setup in this project.